client/src/context/pokemon/PokemonState.js

```diff
@@ -58,6 +60,14 @@ const PokemonState = props => {
     pokemon.sprites.front = res.data.sprites.front_default;
     pokemon.sprites.back = res.data.sprites.back_default;
 
+    // Add level: random between 1 and 100
+    const randomLevel = getRandomBetween(1, 100);
+    pokemon.level = randomLevel;
+
+    // Add moves: 2 random from moves array
+    const moves = res.data.moves;
+    pokemon.moves = getRandomMoves(moves, 2);
```

Comment:
Everything is okay here in terms of code, but I think there's a "bug" in the architecture 😄

Right now the flow looks like this:
1. Client (React app) calls Pokeapi, gets pokemon and randomizes level/moves
2. Client sends POST request with randomized Pokemon to your Node.js server
3. Node.js server saves it in database

The problem here is that I can easily send POST requests manually and add Pokemons with max level (is there a validation? 😛) and best moves.

It would be better to move the randomization logic to the server (and also the call to Pokeapi), but you can leave it as it is for now as it works 👍